python/pyspark/sql/dataframe.py

```diff
@@ -1393,6 +1393,16 @@ def replace(self, to_replace, value=None, subset=None):
         |null|  null| null|
         +----+------+-----+
 
+        >>> df4.na.replace('Alice', None).show()
```

Comment:
I filed a JIRA for mismatching default value between `replace` and `na.replace` in SPARK-21658